radicle/src/cob/identity.rs

```diff
@@ -16,7 +16,7 @@ use crate::{
         store::{self, FromHistory as _, Transaction},
     },
     identity::{doc::DocError, Did, Identity, IdentityError},
-    prelude::Doc,
+    prelude::{Doc, ReadRepository},
```

Comment:
What about `prelude::*`?  Prelude's should be the special import where we import everything no?

Reply:
We could treat it that way if we wanted :shrug: